bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/StateMachine.java

```diff
@@ -18,10 +18,10 @@
 
 package io.ballerina.runtime.internal;
 
+import io.ballerina.identifier.Utils;
 import io.ballerina.runtime.api.creators.ErrorCreator;
 import io.ballerina.runtime.api.utils.StringUtils;
 import io.ballerina.runtime.api.values.BError;
-import org.apache.commons.lang3.StringEscapeUtils;
```

Comment:
The `io.ballerina.identifier.Utils` class uses `StringEscapeUtils.unescapeJava`. We can replace that with `Utils.unescapeJava`.

Reply:
Mistake from me. Within util class, we use `unescapeJava`